hack/generate-manifests-tree.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+OPERATOR_IMAGE="${1:-REPLACE_IMAGE}"
+OUT_DIR="build/_output/manifests"
+
+rm -rf "${OUT_DIR}"
+mkdir -p "${OUT_DIR}"
+cp -a deploy/olm-catalog/performance-addon-operator "${OUT_DIR}"
+find "${OUT_DIR}" -type f -exec sed -i "s|REPLACE_IMAGE|${OPERATOR_IMAGE}|g" {} \; || :
```

Comment:
I see you default to REPLACE_IMAGE -> REPLACE_IMAGE replacement. Is that robust enough? Would maybe a check based on $# be better?

Reply:
Considering the previous comment, it is probably better to exit with error from this script if the argument is missing.